compose.yml

```diff
@@ -28,6 +28,13 @@ services:
         - SOURCE_DATABASE_URL=postgresql://postgres:password@odyssey-timescale:5432/postgres
        # - PROD_SIREN_HOST_URL=siren:1883
         - SCYLLA_PROD=true
+        #- SCYLLA_SATURATE_BATCH=false
+        #-SCYLLA_DATA_UPLOAD_DISABLE=false
+        #-SCYLLA_SIREN_HOST_URL=localhost:1883
+        #-SCYLLA_BATCH_UPSERT_TIME=10
+        - SCYLLA_RATE_LIMIT_MODE=static
+        - SCYLLA_STATIC_RATE_LIMIT_VALUE=50
+        #-SCYLLA_SOCKET_DISCARD_PERCENT=0
```

Comment:
why all this commented out?

Reply:
this is just a way for me to document the settings without opening the code.

Reply:
heard, sounds good